apps/browser/src/background/main.background.ts

```diff
@@ -201,6 +203,7 @@ export default class MainBackground {
   configApiService: ConfigApiServiceAbstraction;
   devicesApiService: DevicesApiServiceAbstraction;
   deviceTrustCryptoService: DeviceTrustCryptoServiceAbstraction;
+  authReqCryptoService: AuthRequestCryptoServiceAbstraction;
```

Comment:
nit: don't abbreviate, makes searching harder, breaks conventions arbitrarily (since we might choose different abbreviations in different parts of the code), etc.
```suggestion
  authRequestCryptoService: AuthRequestCryptoServiceAbstraction;
```